cli/bpmetadata/markdown.go

```diff
@@ -96,3 +112,53 @@ func getMdContent(content []byte, headLevel int, headOrder int, headTitle string
 
 	return nil, fmt.Errorf("unable to find md content")
 }
+
+// getDeploymentDuration creates the deployment and configuration time
+// estimates for the blueprint from README.md
+func getDeploymentDuration(content []byte, headTitle string) (*BlueprintTimeEstimate, error) {
+	durationDetails, err := getMdContent(content, -1, -1, headTitle, true)
+	if err != nil {
+		return nil, err
+	}
+
+	re := regexp.MustCompile(timeEstimateRegexp)
```

Comment:
If we are using `MustCompile`, we can move this to a global var similar to https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/14486905e29e1d4f574a34686a89e54f442636bc/cli/bpmetadata/path.go#L18-L21

Reply:
done